tests/test_enterprise/api/test_views.py

```diff
@@ -8477,3 +8478,17 @@ def test_sso_configuration_delete_not_found(self):
+        entityId = fetch_entity_id_from_metadata_xml(metadata_xml)
+        assert entityId == "https://login.rz.rwth-aachen.de/shibboleth"
+        metadata_xml = '<md:EntitiesDescriptor xmlns:md="urn:oasis:names:tc:SAML:2.0:metadata"> <md:EntityDescriptor entityID="https://idp1.example.com"> <md:IDPSSODescriptor></md:IDPSSODescriptor> </md:EntityDescriptor> <md:EntityDescriptor entityID="https://idp2.example.com"> <md:IDPSSODescriptor></md:IDPSSODescriptor> </md:EntityDescriptor> </md:EntitiesDescriptor>'  # pylint: disable=line-too-long
+        entityId = fetch_entity_id_from_metadata_xml(metadata_xml)
+        assert entityId == "https://idp1.example.com"
```

Comment:
Ideally, a unit test should only test one use-case/scenario per test but if you need to run the same test on different inputs, you can use Python's DDT library to pack test data in tuples. https://ddt.readthedocs.io/en/latest/example.html